src/cart/crt_rpc.h

```diff
@@ -216,7 +216,7 @@ crt_rpc_unlock(struct crt_rpc_priv *rpc_priv)
 	D_MUTEX_UNLOCK(&rpc_priv->crp_mutex);
 }
 
-#define CRT_PROTO_INTERNAL_VERSION 4
+#define CRT_PROTO_INTERNAL_VERSION 5
```

Comment:
Is this proto used client-> server or just server->server?  We're not breaking 2.4 client compatibility here are we?

Reply:
yes, this is just server<->server changes.